src/test/regress/expected/multi_subquery_union.out

```diff
@@ -1237,5 +1237,14 @@ ORDER BY types;
      3 |              1
 (4 rows)
 
+-- Previously this produced a segfault from standard_planner introducing a subquery after we'd called AssignRTEIdentities
+CREATE OR REPLACE FUNCTION users_udf()
+RETURNS TABLE(user_id int)
+AS $$SELECT user_id FROM users_reference_table;$$
+LANGUAGE sql stable;
+SELECT user_id FROM users_table
+UNION SELECT u.user_id FROM users_table, users_udf() u;
+ERROR:  cannot perform distributed planning on this query because parameterized queries for SQL functions referencing distributed tables are not supported
```

Comment:
Why do we hit this error?

Reply:
`distributed_planner` ends up being called on `SELECT user_id FROM public.users_udf() u(user_id)` while planning this query